tests/Microsoft.Identity.Test.Common/TestConstants.cs

```diff
@@ -31,7 +31,7 @@ public static HashSet<string> s_scope
         public static readonly string[] s_graphScopes = new[] { "user.read" };
         public const uint JwtToAadLifetimeInSeconds = 60 * 10; // Ten minutes
         public const string ClientCredentialAudience = "https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/v2.0";
-        public const string AutomationTestThumbprint = "4E87313FD450985A10BC0F14A292859F2DCD6CD3";
+        public const string AutomationTestThumbprint = "444B697D869032F29F9A162D711AF3E2791AD748";

```

Comment:
Is there a reason we don't get the certs by name?

Reply:
It's coming in a separate commit.

Reply:
Due to the time limitation, we are doing it as two-step process. This is the first one. The second is in draft: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3950